submissions/dimonlakhin/html-css-popup/style.css

```diff
@@ -0,0 +1,269 @@
+  list-style-type: none;
+}
+
+.nav-item {
+  margin-right: 30px;
+  font-size: 25px;
+}
+
+.link-flex {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 63px;
+  height: 63px;
+}
+
+.link-flex > img {
+  margin: 0;
+}
+
+.nav-item__icon-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 63px;
+  height: 63px;
+}
+
+.nav-item__link:focus,
+.nav-item__link:hover,
+.popup-link:hover,
+.popup-link:focus,
+label:hover,
+.check__input:focus + label {
+  outline: var(--border);
+  border-radius: var(--border-radius);
+  box-shadow: 0 3px 3px var(--black);
+}
+
+.nav-item__link:active,
+.popup-link:active,
+label:active {
+  border-radius: var(--border-radius);
+  box-shadow: 0 var(--size-5) var(--size-5) var(--black);
+}
+
+.nav-item__link {
+  color: var(--black);
+  text-decoration: none;
+}
+
+.nav-item__link span {
+  padding: 8px;
+}
+
+.more_label {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 120px;
+  font-size: var(--size-20);
+}
+
+.nav-item__icon {
+  width: 50px;
+  margin-top: 5px;
+  border-radius: 50%;
+  border: var(--border);
+  color: var(--black);
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.check__input {
+  -webkit-appearance: none;
+}
+
+.check__input:checked ~ .popup {
+  display: flex;
+}
+
+.check__input:checked ~ .container-3 {
+  display: flex;
+}
+
+.popup {
+  display: none;
+  position: absolute;
+  top: var(--default-height);
+  left: -120px;
+  width: 300px;
+  height: 400px;
+  padding-bottom: var(--size-10);
+  flex-direction: column;
+  overflow-y: auto;
+  overflow-x: hidden;
+  background-color: rgb(253, 239, 207);
+  border: var(--border);
+  border-radius: 8px;
+  scrollbar-gutter: stable;
+  transition: scrollbar-color 0.3s ease-out;
+}
+
+.popup::-webkit-scrollbar {
+  width: var(--size-10);
+}
+
+.popup::-webkit-scrollbar-track {
+  background-color: transparent;
+}
+
+.popup::-webkit-scrollbar-thumb {
+  background: var(--orange);
+  border-radius: var(--border-radius);
+  box-shadow: inset 0 0 6px var(--orange);
+}
+
+.popup-link {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  row-gap: 3px;
+  margin-top: var(--size-5);
+  color: var(--black);
+  text-decoration: none;
+}
+
+.container-2 {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  flex-wrap: wrap;
+  row-gap: var(--size-10);
+  list-style-type: none;
+}
+
+.container-3 {
+  display: none;
+  justify-content: space-evenly;
+  align-items: center;
+  flex-wrap: wrap;
+  row-gap: var(--size-10);
+  list-style-type: none;
+  font-size: var(--size-20);
+}
+
+.nav-item-2,
+.nav-item-3 {
+  width: 120px;
+  font-size: var(--size-20);
+}
+
+.kottans-link {
+  width: 80%;
+  margin: auto;
+  padding-top: var(--size-10);
+  padding-left: 7px;
+}
+
+.kottans-link a {
+  padding: var(--size-5);
+}
+
+label {
```

Comment:
Please, avoid to style by HTML tags, read more [here](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)